string[] names
float32[] stiffnesses
---
bool res